platform_versioned_docs/version-23.4.0/data/data-studios.mdx

```diff
@@ -102,31 +108,53 @@ Once connected, the data studio session list will update to display the status o
 
 ![](./_images/data_studios.png)
 
-### Collaborate in a data studio
+## Collaborate in a data studio
+
+This functionality is available to all user roles excluding the **View** role.
 
 To share a link to a running data studio with collaborators inside your workspace, select the three dots next to the status message for the data studio you want to share, then select **Copy data studio URL**. Using this link other authenticated users can access the session directly.
 
 :::note
-Collaborators will need valid permissions to connect to the running data studio (at least the **Connect** role for the workspace).
+Collaborators need valid workspace permissions to connect to the running data studio.
 :::
 
-### Stop a data studio
+## Stop a data studio
 
-To stop a running session, select the three dots next to the status message and then select **Stop**. The status will change from **running** to **stopped**. When a data studio session is stopped, the compute resources it's using are deallocated and any unsaved state is lost. You can stop a session at any time.
+This functionality is available to all user roles with the **Maintain** role and above.
+
+To stop a running session, select the three dots next to the status message and then select **Stop**. The status will change from **running** to **stopped**. When a data studio session is stopped, the compute resources it's using are deallocated. You can stop a session at any time.
+
+Stopping a running session creates a new checkpoint.
 
 :::note
 Make sure you save before you stop a data studio.
 :::
 
-### Restart a stopped data studio
+## Restart a stopped data studio
+
+This functionality is available to all user roles with the **Maintain** role and above.
+
+When you restart a stopped session, a new session with the same configuration is created, using the most recent checkpoint.
```

Comment:
I'm a bit confused here with what we mean by restart :sweat_smile:. Is this just starting a session or are we referring to start as new from the list?

Reply:
This is just starting a session, not `Start as new`

Reply:
In that case, is there a reason for this section at all considering that there is a section for [Start a data studio](https://github.com/seqeralabs/docs/pull/26/files#diff-fe51911e951cfe92d20f903be2226b5c217b4d737d5904b4ffa83e6933c6a078R85) and a section for [Start a new data studio from a checkpoint](https://github.com/seqeralabs/docs/pull/26/files#diff-fe51911e951cfe92d20f903be2226b5c217b4d737d5904b4ffa83e6933c6a078R139)